app/scripts/lib/setupSentry.js

```diff
@@ -102,6 +102,8 @@ export const SENTRY_BACKGROUND_STATE = {
     showProductTour: true,
     showNetworkBanner: true,
     showAccountBanner: true,
+    switchedNetworkDetails: null,
```

Comment:
This isn't a valid value for the mask. Either use `false` to exclude this, or `true` to include it, or use an object if this is an object that you want to partially include